File: angular/src/app/Components/Lawyer/lawyers/lawyer.ts
import { Case } from "../../Cases/case";

export interface Lawyer {
    id: string;
    name: string;
    position:string
    mobile:string;
    address:string
    cases: Case[];
  }
      
  export interface LawyerCreate {
       id:string;
       name:string; 
       Position:string;
       Mobile:string;
       Address:string 
  }
  

File: aspnet-core/src/Inva.LawMax.Domain/Lawyers/Lawyer.cs
using Inva.LawMax.LawyerCases;
using System;
using System.Collections.Generic;
using Volo.Abp.Domain.Entities.Auditing;

namespace Inva.LawMax.Lawyers
{
    public class Lawyer : FullAuditedAggregateRoot<Guid>
    {
        public string Name { get; set; }
        public string Position { get; set; }
        public string Mobile { get; set; }
        public string Address { get; set; }

        public ICollection<LawyerCase> LawyerCases { get; set; }

        public Lawyer()
        {
            LawyerCases = new List<LawyerCase>();
        }
    }
}
